py/server/deephaven/experimental/data_index.py

```diff
@@ -55,7 +55,7 @@ def has_data_index(table: Table, key_cols: List[str]) -> bool:
     return _JDataIndexer.hasDataIndex(table.j_table, key_cols)
 
 
-def get_data_index(table: Table, key_cols: List[str]) -> Optional[DataIndex]:
+def _get_data_index(table: Table, key_cols: List[str]) -> Optional[DataIndex]:
```

Comment:
I don't think this method is still necessary, but it isn't in the public API, so I don't have a strong view.